manifests/install.pp

```diff
@@ -30,11 +30,26 @@
     extract_path => $aerospike::download_dir,
     creates      => $dest,
     cleanup      => $aerospike::remove_archive,
-  } ~>
-  exec { 'aerospike-install-server':
-    command     => "${dest}/asinstall",
-    cwd         => $dest,
-    refreshonly => true,
+    notify       => Exec['aerospike-install-server'],
```

Comment:
It's better to have reinstall than failed installation. The `~>` arrow have [the same semantic as `notify`](https://docs.puppet.com/puppet/latest/lang_relationships.html#syntax-chaining-arrows), so from this perspective this PR doesn't bring any change at all.